src/pages/Opportunities.jsx

```diff
@@ -0,0 +1,229 @@
+          <input
+            type="text"
+            placeholder="Default Location"
+            className="border rounded-full px-6 py-4 w-full md:w-1/2 mb-4"
+          />
+          <div>
+            {buttons.map((button, index) => (
+              <button
+                key={index}
+                type="button"
+                onClick={() => handleButtonClick(button.text)}
+                className={`font-medium rounded-full text-sm px-5 py-2.5 text-center mr-2 mb-2 
+                  ${
+                    button.text === "All Issues" &&
+                    activeCategories.length === 0
+                      ? "text-white bg-slate-500 hover:bg-slate-400"
+                      : activeCategories.includes(button.text)
+                      ? "text-white bg-slate-500 hover:bg-slate-400"
+                      : "hover:bg-slate-400 focus:ring-blue-300 border border-slate-400"
+                  }`}
+              >
+                {button.text}
+              </button>
+            ))}
+          </div>
+        </div>
+      </section>
+
+      <div className="container flex mx-auto max-w-5xl px-4 py-6 justify-between items-start space-x-8">
+        <div className="w-full md:w-2/3">
+          <div className="mb-6">
+            <h2 className="text-2xl font-extrabold">
+              482 volunteers are needed
+            </h2>
+            <span className="text-slate-500">Showing 1-10</span>
+          </div>
+
+          {opportunitiesList.map((el, index) => (
+            <div className="flex mb-8 border-b pb-8" key={index}>
+              <div className="mr-4">
+                <svg
+                  fill="none"
+                  stroke="currentColor"
+                  viewBox="0 0 200 200"
+                  preserveAspectRatio="none"
+                  aria-hidden="true"
+                  className="border w-32 text-gray-300"
+                >
+                  <path
+                    d="M0 0l200 200M0 200L200 0"
+                    strokeWidth="1"
+                    vector-effect="non-scaling-stroke"
+                  ></path>
+                </svg>
+              </div>
+              <div>
+                <h4>
+                  <a
+                    href={`/opportunities/${el.id}`}
+                    className="underline text-lg text-blue-500 hover:text-blue-400"
+                  >
+                    {el.title}
+                  </a>
+                </h4>
+                <div className="text-sm text-slate-500">
+                  <span>{el.date}, </span>
+                  <span>{el.location}</span>
+                </div>
+                <p className="mb-2">{el.description}</p>
+                <button className="font-medium rounded-full text-sm px-5 py-2.5 text-center bg-slate-200">
+                  Details
+                </button>
+              </div>
+            </div>
+          ))}
+
+          <div className="flex items-center justify-between">
```

Comment:
Add a comment that pagination isn't implemented, and create a Jira ticket to track for future work

Reply:
@justgogame